pkg/image/export/main.go

```diff
@@ -513,7 +513,8 @@ images="rancher-images.tar.gz"
 usage () {
     echo "USAGE: $0 [--image-list rancher-images.txt] [--images rancher-images.tar.gz]"
     echo "  [-l|--image-list path] text file with list of images; one image per line."
-    echo "  [-i|--images path] tar.gz generated by docker save."
+		echo "  [-i|--images path] tar.gz generated by docker save."
```

Comment:
Incorrect use of tabs/spaces. Please correct.

Reply:
This is fixed (I had to go into Atom and fix Soft Tabs.